### PR TITLE
added the youtuber added name, category birthday and views i added th…

### PR DESCRIPTION
**Pull request by Gallantfarhan**
I added a new row to the table with information about a new YouTube channel that meets the requirements for inclusion in the list. The channel has over 100K subscribers and has uploaded a video within the last month. Additionally, the channel is run by an individual rather than a company. The new row includes the category, channel name, birthday, number of views.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -305,6 +305,13 @@ <h1>YouTubers' Birthdays</h1>
 					<td>Janurary 17, 1995</td>
 					<td>147 million</td>
 				</tr>
+		<!-- coding -->
+				<tr>
+					<td>Web development</td>
```
Comment (KendallDoesCoding) [inline index.html:310]: Should we say Web Development or Tech? What do you think @Gallantfarhan?

Comment (Gallantfarhan) [reply index.html:310]: Web Development sounds perfect!

**Comment by KendallDoesCoding** [comment]
@Gallantfarhan - thanks for your contribution!

**Comment by Gallantfarhan** [comment]
your most welcome, I loved contributing to your project.

**Pull request merged**